lang/de.yml

```diff
@@ -180,6 +180,7 @@ de:
     RESPONSE: "Re: %s"
     SHOWLINK: "Zeige Thema"
     SINGULARNAME: "Beitrag"
+    NEWREPLY: "Neue Antwort zu"
```

Comment:
Don't seem to be able to get into transifex :( any change you can add the item?) What would have to happen next after this key is added in transifex?